storybookjs/storybook
# CLI: Remove watch mode from the build-storybook documentation

Issue: https://github.com/storybookjs/storybook/issues/15946

`-w` or `--watch-mode` flag doesn't do anything when running the `build-storybook` script.

## What I did
- Removed `--watch-mode` from commander so that it doesn't show up in the list of available options.
- Removed watch mode from the docs.
## How to test
Run `yarn build-storybook -h`. `--watch-mode` should no longer show up as an option.


If your answer is yes to any of these, please make sure to include it in your PR.
